server/mldb_server.cc

```diff
@@ -187,26 +188,49 @@ initRoutes()
                          handleShutdown,
                          Json::Value());
 
-    addRouteAsync(versionNode, "/query", { "GET" },
-                  "Select from dataset",
-                  &MldbServer::runHttpQuery,
-                  this,
-                  RestParam<Utf8String>("q", "The SQL query string"),
-                  PassConnectionId(),
-                  RestParamDefault<std::string>("format",
-                                                "Format of output",
-                                                "full"),
-                  RestParamDefault<bool>("headers",
-                                         "Do we include headers on table format",
-                                         true),
-                  RestParamDefault<bool>("rowNames",
-                                         "Do we include row names in output",
-                                         true),
-                  RestParamDefault<bool>("rowHashes",
-                                         "Do we include row hashes in output",
-                                         false));
+
+   // MLDB-1380 - make sure that the CPU support the minimal instruction sets
+    if (ML::has_sse42()) {
```

Comment:
Consider splitting this out into a separate function like supportsSystemRequirements()